addons/hr_expense/models/hr_expense.py

```diff
@@ -217,17 +243,39 @@ def _compute_is_ref_editable(self):
             else:
                 expense.is_ref_editable = is_account_manager
 
+    @api.depends_context('lang')
+    @api.depends('product_id')
+    def _compute_product_description(self):
+        self.product_description = ""
+        for expense in self:
+            if not expense.product_id:
+                expense.product_description = False
+            else:
+                expense.product_description = expense.product_id.description if not is_html_empty(html_sanitize(expense.product_id.description, strip_style=True, strip_classes=True)) else False
```

Comment:
Like already raised by @csnauwaert , this is basically a related field, lets have it look like one

```suggestion
        for expense in self:
            expense.product_description = not is_html_empty(expense.product_id.description) and expense.product_id.description
```

`html_sanitize` shouldn't be needed with the `Markup` object, and the method `is_html_empty` was designed just for this (look at the docstring)
no need to set `self.product_description = ""` if you are setting it on all records afterwards anyways
`expense.product_id.description` is `False` if there is no product anyway

Reply:
Very correct point indeed, thanks :raised_hands: